test/test_scanner.py

```diff
@@ -192,6 +192,18 @@ def test_rpm_sqlite_3_12_2(self):
         'sqlite',
         '3.12.2')
 
+    @unittest.skipUnless(os.getenv('LONG_TESTS') == '1', 'Skipping long tests')
+    def test_rpm_xml2_2_9_1(self):
+        """
+        test to see if we detect a real copy of libxml2 2.9.1
+        as opposite to the faked binary used for the fast tests
+        """
+        self._file_test(
+	'http://mirror.centos.org/centos/7/os/x86_64/Packages/',
```

Comment:
Please fix the indent to be pylint-friendly.  (We don't have it enabled in Travis yet, but I'm hoping to in the future.)